content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -1,325 +1,546 @@
+Name:                   object-counts
+Namespace:              myspace
+Resource                Used    Hard
+--------                ----    ----
+configmaps              0       10
+persistentvolumeclaims  0       4
+replicationcontrollers  0       20
+secrets                 1       10
+services                0       10
+services.loadbalancers  0       2
+```
+
+Kubectl also supports object count quota for all standard namespaced resources
+using the syntax `count/<resource>.<group>`:
+
+```shell
+kubectl create namespace myspace
+
+kubectl create quota test --hard=count/deployments.extensions=2,count/replicasets.extensions=4,count/pods=3,count/secrets=4 --namespace=myspace
+
+kubectl run nginx --image=nginx --replicas=2 --namespace=myspace
+
+kubectl describe quota --namespace=myspace
+Name:                         test
+Namespace:                    myspace
+Resource                      Used  Hard
+--------                      ----  ----
+count/deployments.extensions  1     2
+count/pods                    2     3
+count/replicasets.extensions  1     4
+count/secrets                 1     4
+```
+
+## Quota and Cluster Capacity
+
+`ResourceQuotas` are independent of the cluster capacity. They are
+expressed in absolute units.  So, if you add nodes to your cluster, this does *not*
+automatically give each namespace the ability to consume more resources.
+
+Sometimes more complex policies may be desired, such as:
+
+  - Proportionally divide total cluster resources among several teams.
+  - Allow each tenant to grow resource usage as needed, but have a generous
+    limit to prevent accidental resource exhaustion.
+  - Detect demand from one namespace, add nodes, and increase quota.
+
+Such policies could be implemented using `ResourceQuotas` as building blocks, by
+writing a "controller" that watches the quota usage and adjusts the quota
+hard limits of each namespace according to other signals.
+
+Note that resource quota divides up aggregate cluster resources, but it creates no
+restrictions around nodes: pods from several namespaces may run on the same node.
+
+## Limit Priority Class consumption by default
+
+It may be desired that pods at a particular priority, eg. "cluster-services", should be allowed in a namespace, if and only if, a matching quota object exists.
+
+With this mechanism, operators will be able to restrict usage of certain high priority classes to a limited number of namespaces and not every namespaces will be able to consume these priority classes by default.
+
+To enforce this, kube-apiserver flag `--admission-control-config-file` should be used to pass path to the following configuration file:
+
+```shell
+$ cat admission_config_file.yml
+apiVersion: apiserver.k8s.io/v1alpha1
+kind: AdmissionConfiguration
+plugins:
+- name: "ResourceQuota"
+  configuration:
+    apiVersion: resourcequota.admission.k8s.io/v1alpha1
+    kind: Configuration
+    limitedResources:
+    - resource: pods
+    matchScopes:

```

Comment:
The indentation is off here. The "matchScopes" field should be indented as a sibling of the "resource" field above, per [the `LimitedResource` struct](https://github.com/kubernetes/kubernetes/blob/d4d02762340bfb6ac93afd10884a3d9941b7322f/plugin/pkg/admission/resourcequota/apis/resourcequota/types.go#L38-L72).

Reply:
See #12055.